service_actor/app/repositories/dao/ActorDao.scala

```diff
@@ -47,10 +47,10 @@ class ActorDao @Inject() (
     db.run(query.result.headOption)
   }
 
-  def getByName(searchString: String): Future[Seq[Person]] = {
+  def getByName(mid: MuseumId, searchString: String): Future[Seq[Person]] = {
     val likeArg = searchString.toLowerCase
     val query = actorTable.filter { a =>
-      a.fn.toLowerCase like s"%$likeArg%"
+      (a.fn.toLowerCase like s"%$likeArg%") && a.museumId === mid.underlying
```

Comment:
You can remove the `underlying` call by changing the setup of the table to read the Int to a `MuseumId` (line 82 in this file). You also need to add a Column type for `MuseumId` in `ColumnTypeMappers`